Cubical/Structures/AbGroup.agda

```diff
@@ -0,0 +1,47 @@
+{-# OPTIONS --cubical --safe #-}
+module Cubical.Structures.AbGroup where
+
+open import Cubical.Foundations.Prelude
+open import Cubical.Foundations.Equiv
+open import Cubical.Foundations.HLevels
+open import Cubical.Data.Sigma
+open import Cubical.Foundations.SIP renaming (SNS-PathP to SNS)
+
+open import Cubical.Structures.NAryOp
+open import Cubical.Structures.Group
+
+private
+  variable
+    ℓ : Level
+
+raw-abelian-group-structure : Type ℓ → Type ℓ
```

Comment:
I wouldn't introduce this and instead just use `raw-group-structure`, the same holds for some of the other definitions below. Abelian groups should just be introduced by adding the commutativity axiom to groups

Reply:
Totally agree on this, I first thought I would be better to have each structure defined in the exact same style but thanks to your comment I totally changed my mind, I'm going to fix it.

Reply:
I changed the only raw-abelian-group-structure for raw-group-structure, should I change both for raw-semigroup-structure (in case we finally stick to the actual raw-group-structure)?